app/src/main/java/com/manuel28g/carsales/covidworlddata/ui/fragment/HomeFragment.kt

```diff
@@ -71,4 +84,7 @@ class HomeFragment: Fragment(), DatePickerDialog.OnDateSetListener{
         findNavController().navigate(R.id.action_home_to_error)
     }
 
+    private fun setupInjection(){
+        (context?.applicationContext as AppCovid).getComponent().inject(this)
```

Comment:
Just as an improvement. You can create an extension called **appComponent()** that returns the dagger component you need 👍